clean.cmd

```diff
@@ -0,0 +1,105 @@
+@echo off
+setlocal
+
+set cleanlog=%~dp0clean.log
+echo Running Clean.cmd %* > %cleanlog%
+
+if [%1] == [] (
+  set clean_targets=Clean
+  goto Begin
+)
+
+set clean_targets=
+set clean_src=
+set clean_tools=
+set clean_all=
+
+:Loop
+if [%1] == [] goto Begin
+
+if /I [%1] == [/?] goto Usage
+
+if /I [%1] == [/b] (
+  set clean_targets=Clean;%clean_targets%
+  goto Next
+)
+
+if /I [%1] == [/p] (
+  set clean_targets=CleanPackages;%clean_targets%
+  goto Next
+)
+
+if /I [%1] == [/c] (
+  set clean_targets=CleanPackagesCache;%clean_targets%
+  goto Next
+)
+
+if /I [%1] == [/s] (
+  set clean_src=true
+  goto Next
+)
+
+if /I [%1] == [/t] (
+  set clean_tools=true
+  goto Next
+)
+
+if /I [%1] == [/all] (
+  set clean_src=
+  set clean_tools=
+  set clean_targets=Clean;CleanPackages;CleanPackagesCache;
```

Comment:
Would setting clean_targets to "CleanPackagesCache" be sufficient?  The "git clean -xdf" command will delete bin and packages from the repo.